packages/story-editor/src/components/footer/layers/layers.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import { useState, useRef } from '@googleforcreators/react';
+import { __ } from '@googleforcreators/i18n';
+/**
+ * Internal dependencies
+ */
+import { Z_INDEX_FOOTER } from '../../../constants/zIndex';
+import { LayerPanel } from '../../panels/design';
+import useLayers from '../../panels/design/layer/useLayers';
+import Popup, { NavigationWrapper } from '../../secondaryPopup';
+import { ToggleButton } from '../../toggleButton';
+
+const LAYERS_PANEL_WIDTH = 300;
+
+const StyledNavigationWrapper = styled(NavigationWrapper)`
+  width: ${LAYERS_PANEL_WIDTH}px;
+  left: -${LAYERS_PANEL_WIDTH}px;
```

Comment:
`SecondaryPopup` was created with the left (in LTR) side of the footer in mind, so everything is aligned to the left. This is quick and dirty way to align to the right side. It's the only one that does this so I'd rather not dirty up the shared popup for 1 case